src/System.Diagnostics.Process/src/System/Diagnostics/AsyncStreamReader.cs

```diff
@@ -215,9 +156,7 @@ private void ReadBuffer(Task<int> t)
         // immediately followed by a line feed. The resulting string does not
         // contain the terminating carriage return and/or line feed. The returned
         // value is null if the end of the input stream has been reached.
-        //
-
-        private void GetLinesFromStringBuilder()
+        private void GetLinesFromStringBuilder(CancellationToken token)
```

Comment:
Could we rename this method?  I find the existing name to be less than explanatory.  How about MoveLinesFromStringBuilderToMessageQueue?